tools/run_tests/run_interop_tests.py

```diff
@@ -1042,6 +1042,7 @@ def aggregate_http2_results(stdout):
 # Start interop servers.
 server_jobs = {}
 server_addresses = {}
+num_failures = 0
```

Comment:
I went back to the first version of this file and it didn't have sys.exit after the test run. There are sys.exits for other kinds of failures in the file though.